src/concrete/ml/sklearn/base.py

```diff
@@ -565,7 +564,7 @@ def compile(
         assert isinstance(self.fhe_circuit, Circuit)
 
         # CRT simulation is not supported yet
-        # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/3841
+        # TODO: https://github.com/zama-ai/concrete-ml-internal/issues/3841
```

Comment:
we prefer to keep everything as a FIXME ! Most of our scripts only look at fixme's and not todo's

Reply:
I didn't change it, I thought it appeared during the rebase.
I'm going to change it.

Reply:
yes I recently changed todos to fixme, you might have just accepted the wrong change when rebasing indeed !